qiskit/test/base.py

```diff
@@ -144,6 +145,24 @@ def assertDictAlmostEqual(
             msg = self._formatMessage(msg, error_msg)
             raise self.failureException(msg)
 
+    def enableParallelProcessing(self):
```

Comment:
It looks like we disable the `invalid-name` rule for pylint at [the top of the file](https://github.com/Qiskit/qiskit-terra/blob/1658bb4626ad21d31fa0403f80f20b4626d74b98/qiskit/test/base.py#L13). This is probably so that it doesn't spit out warnings for functions such as `setUp` and `tearDown` for which we don't have control of the naming scheme. The function has been renamed in [c1d0fe7](https://github.com/Qiskit/qiskit-terra/pull/8110/commits/c1d0fe7af3d6ad44eab9d19c57992892a4564de6).